Jenkinsfile

```diff
@@ -33,6 +33,42 @@ pipeline {
                 }
             }
         }
+        stage("Test sparklyr on Databricks cluster (not Databricks Connect)") {
+            steps {
+                script {
+                    def repo = sh(script: "git remote get-url origin | cut -d/ -f 4,5", returnStdout: true).trim()
+                    print("REPO: " + repo)
```

Comment:
Should we print these? If so, maybe print in a way that shows up in the log file as well.